docs/machine-learning/tutorials/sentiment-analysis-model-builder.md

```diff
@@ -60,63 +54,68 @@ Each row in the *wikipedia-detox-250-line-data.tsv* dataset represents a differe
 1 | == OK! ==  IM GOING TO VANDALIZE WILD ONES WIKI THEN!!!
 0 | I hope this helps.
 
+## Create a Model Builder config file
+
+When first adding a machine learning model to the solution it will prompt you to create an `mbconfig` file. The `mbconfig` file keeps track of everything you do in Model Builder to allow you to reopen the session.
+
+1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
+1. In the dialog, name the Model Builder project **SentimentAnalysis**, and click **Add**.
+
 ## Choose a scenario
 
 ![Model Builder wizard in Visual Studio](../media/model-builder-scenarios.png)
 
 To train your model, you need to select from the list of available machine learning scenarios provided by Model Builder.
 
-1. In **Solution Explorer**, right-click the *SentimentRazor* project, and select **Add** > **Machine Learning Model...**.
-1. For this sample, the scenario is sentiment analysis. In the *scenario* step of the Model Builder tool, select the **Sentiment Analysis** scenario.
+1. For this sample, the task is text classification. In the *Scenario* step of the Model Builder extension, select the **Text classification** scenario.
+
+## Select an environment
+
+Model Builder can run the training on different environments depending on the scenario that was selected.
+
+Select **Local (GPU)** as your environment and click the **Next step** button.
+
+> [!NOTE]
+> This scenario uses deep learning techniques which work best in GPU environments. If you don't have a GPU, choose the Local (CPU) environment but note that the expected time to train will be significantly longer. For more information on using GPUs with Model Builder, see the [GPU support in Model Builder guide](../how-to-guides/install-gpu-model-builder.md).
```

Comment:
are you able to add the link for environment selection for further details here ?

Reply:
What's that link?

Reply:
https://learn.microsoft.com/en-us/dotnet/machine-learning/automate-training-with-model-builder#environment

Reply:
@zewditu Sorry. I'm not sure what the ask is here. Is it that I add this link somewhere in the doc or that we link to it from Model Builder?